recipe/meta.yaml

```diff
@@ -1,26 +1,25 @@
 {% set name = "catalystcoop.pudl" %}
-{% set version = "2023.12.1" %}
+{% set version = "2024.2.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 17c75946a35f65c8a96f042b55337e54f8836acb43c00b6c1c38b82c0390ffcd
+  sha256: ef80272341f22ebf9077501ddf5c886b1f157a9b455b653290c567067d70126c
 
 build:
   noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
   entry_points:
-    - metadata_to_rst = pudl.convert.metadata_to_rst:main
-    - epacems_to_parquet = pudl.convert.epacems_to_parquet:main
     - ferc_to_sqlite = pudl.ferc_to_sqlite.cli:main
+    - metadata_to_rst = pudl.convert.metadata_to_rst:main
+    - pudl_check_fks = pudl.etl.check_foreign_keys:pudl_check_fks
     - pudl_datastore = pudl.workspace.datastore:main
     - pudl_etl = pudl.cli.etl:main
-    - pudl_setup = pudl.workspace.setup_cli:main
-    - state_demand = pudl.analysis.state_demand:main
+    - pudl_service_territories = pudl.analysis.service_territory:pudl_service_territories
```

Comment:
I alphabetized these and updated the list to reflect those entrypoints defined in `pyproject.toml`